### ISSUE TITLE
Support generation on CI

### ISSUE THREAD

**Issue by kettanaito** [kettanaito/remix-og-image#6]
Most CI providers won't run Puppeteer for one reason or the other:

- Vercel: Puppeteer's binary size exceeds the function limit size of 50MB (may be irrelevant for the build).
- Cloudflare: No Puppeteer support, to begin with. Have their mirror package `@cloudflare/puppeteer` locked behind a paid Browser mode feature. 

However, some hosting providers can support Puppeteer through their own bindings and workarounds. Usually, those revolve around using a different Chromium binary and sometimes a different Puppeteer package altogether (like my [find on how to run Puppeteer on Vercel](https://gist.github.com/kettanaito/56861aff96e6debc575d522dd03e5725)).

The plugin can expose options to support custom binaries or Puppeteer packages to enable image generation during the CI builds. This would be amazing.

## Syntax

Here's a made-up example of hooking AWS Chromium to this plugin. This usually unblocks Puppeteer on Vercel, but it also requires to use a specific version of `puppeteer-core` to comply with the file size limit. 

```js
import { edgeChromium } from 'chrome-aws-labmda'

openGraphImagePlugin({
  browser: isCI && {
    executablePath: edgeChromium.executablePath,
    args: edgeChromium.args
  },
})
```

**Comment by kettanaito** [comment]
Exposing the Puppeteer customization may not be necessary. Hosting providers run builds in Docker containers where you can have Chromium installed. Needs verification and examples.

**Comment by kettanaito** [comment]
## Released: v0.3.4 🎉

This has been released in v0.3.4!

- 📄 [**Release notes**](https://github.com/kettanaito/remix-og-image/releases/tag/v0.3.4)
- 📦 [npm package](https://www.npmjs.com/package/remix-og-image/v/0.3.4)

Make sure to always update to the latest version (`npm i remix-og-image@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
